Portenta X8: Add image target version advise[PC-921] (#296)
arduino/docs-content: content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md

* Wi-Fi AP with Internet access.
* Arduino Pro Cloud Subscription. [Learn more about the Pro Cloud](https://www.arduino.cc/pro/hardware/product/portenta-x8#pro-cloud).

> ***Attention: We encourage you to check frequently if the device image version is up to date in order to have the latest bootloader and patches for bugs, please check the tutorial [How to flash your Portenta X8](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing) in order to update it.***

Reviewer: ```suggestion
***Attention: We encourage you to check every now and then if the device image version is up to date in order to have the latest bootloader, please check the tutorial [How to flash your Portenta X8](https://docs.arduino.cc/tutorials/portenta-x8/image-flashing) for instructions.***
```
